packages/backend/package.json

```diff
@@ -4,7 +4,7 @@
     "lint": "yarn lint:local",
     "format": "yarn format:local",
     "format:check": "yarn format:check:local",
-    "test": "yarn test:local",
+    "test": "yarn test:local --runInBand",
```

Comment:
@wilsonianb this flag might help make the tests more consistent for debugging https://github.com/coilhq/rafiki/pull/44#issuecomment-875102781